microcosm_daemon/daemon.py

```diff
@@ -113,7 +113,9 @@ def run(self):
 
         if args.processes < 1:
             parser.error("--processes must be positive")
-        elif args.processes == 1:
+        elif args.processes == 1 and args.heartbeat_threshold_seconds < 0:
```

Comment:
This check wasn't what I was expecting, but the off value/default for `heartbeat_threshold_seconds` seems to be -1